packages/mobile/src/app/TrpcProvider.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { useMemo, useState } from 'react'
+
+import { accountSelectors, createAudiusTRPCClient, trpc } from '@audius/common'
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query'
+import { useSelector } from 'react-redux'
+
+export const AudiusTrpcProvider = ({
+  children
+}: {
+  children: React.ReactNode
+}) => {
+  const currentUserId = useSelector(accountSelectors.getUserId)
+  const [queryClient] = useState(() => new QueryClient())
+  const trpcClient = useMemo(
+    () => createAudiusTRPCClient(currentUserId),
```

Comment:
We have all caps `TRPC` here and camel `Trpc` in `AudiusTrpcProvider`, let's standardize on one. I don't actually know what the standard is in the rest of the codebase, I see both `Http` and `API` 😢

I personally prefer camel

Reply:
I also prefer camel